.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ name: Tests
 env:
     ETHERSCAN_API_KEY_ETHEREUM: I5BXNZYP5GEDWFINGVEZKYIVU2695NPQZB
     ETHERSCAN_API_KEY_CELO: B13XSMUT6Q3Q4WZ5DNQR8RXDBA2KNTMT4M
-    GOERLI_PRIVATE_KEY: "fa4a1a79e869a96fcb42727f75e3232d6865a82ea675bb95de967a7fe6a773b2"
+    SEPOLIA_PRIVATE_KEY: "fa4a1a79e869a96fcb42727f75e3232d6865a82ea675bb95de967a7fe6a773b2"
```

Comment:
let's keep goerli as well

we could also switch to gh secrets

Reply:
do you mean we shouldn't change to sepolia at all, or just keep the env var name as goerli?

Reply:
keep both

Reply:
problem is that tests are flaky with goerli